repohandler.go

```diff
@@ -0,0 +1,13 @@
+package repohandler
+
+//  Repohandler is a general interface
+//  for working with database
+
+// RepoHandler ...
+type RepoHandler interface {
```

Comment:
Each handler should come from each interface

Reply:
Not quite sure what you meant bhaiya @aukgit